LambdaEngine/Source/Resources/AnimationGraph.cpp

```diff
@@ -37,7 +37,10 @@ namespace LambdaEngine
 	*/
 
 	Transition::Transition(const String& fromState, const String& toState, float64 beginAt)
-		: m_IsActive(false)
+		: m_pOwnerGraph(nullptr)
+		, m_From(0xffffffff)
+		, m_To(0xffffffff)
```

Comment:
Again, if you decide to use `UINT32_MAX`, here's another place to do some replacing.